### PR TITLE
Log Files on Pantheon 

### LINKED ISSUE #1849: Debugging Sites with Log Files Doc Update
Re: [Debugging Sites with Log Files](https://pantheon.io/docs/debug-log-files)
## Issue Description:
- Most valuable content is within the FAQ section at the bottom. 
- WP section is unrelated to logs.

## Suggested Resolution 
Consider moving FAQs to the main the Log doc or refactoring this doc for improvement. 

cc @nataliejeremy @ari-gold

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #1849 

## Effect
PR includes the following changes:
- Moves relevant content from debug-log-files to /docs/logs
- Updates links from /docs/debug-log-files to /docs/logs
- Deletes /docs/debug-log-files

## To Do
- [x] Add redirect from /docs/debug-log-files to /docs/logs

@rachelwhitton @ari-gold Please review

### CODE REVIEW

File: source/_docs/logs.md [hunk 1]
```diff
@@ -147,9 +147,74 @@ sh collect-logs.sh
...
+```bash
+terminus drush "watchdog-show"
+```
+
+Terminus can invoke Drush commands to "watch" events in real-time; tail can be used to continuously show new watchdog messages until interrupted (Control+C).  
+
+```bash
+terminus drush "watchdog-show --tail"
+```
+
```
Comment (rachelwhitton) [inline source/_docs/logs.md]: Suggest adding an FAQ for enabling `WP_DEBUG` and `WP_DEBUG_LOG`, similar to the statement made in [PHP Errors and Exceptions](https://pantheon.io/docs/php-erros) such as:
>**How do I display and log PHP errors or notices for WordPress?** 
>Enable the [WP_DEBUG and WP_DEBUG_LOG](https://codex.wordpress.org/Debugging_in_WordPress) constants to write errors to `wp-content/debug.log` and show all PHP errors, notices and warnings on the page. We suggest setting the WordPress debugging constants per environment:
```
<script src="//gist-it.appspot.com/https://github.com/pantheon-systems/pantheon-settings-examples/blob/master/wordpress/wp-debug-expanded.wp-config.php?footer=minimal"></script>
```

**Comment by rachelwhitton** [comment]
One small suggestion to include WordPress debugging content, otherwise looks great! Thanks @nataliejeremy! 

+1

**Comment by nataliejeremy** [comment]
@ari-gold Mind taking a look? We're ready to deploy this one.

File: source/_docs/logs.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Log files track and record your site's activity to help you find, debug, and iso
         <td>Webserver error log.</td>
       </tr>
       <tr>
-        <th>php-error.log</th>
+        <th>php-error.log <a class="pop" rel="popover" data-proofer-ignore data-toggle="popover" data-html="true" data-content="The PHP error log is provided in each environment on the <strong>Errors</strong> tab of the Site Dashboard. For details, see <a href='/docs/php-errors'>PHP Errors and Exceptions</a>."><em class="fa fa-info-circle"></em></a></th>
         <td>1MB of log data</td>
         <td>PHP <a href="http://php.net/manual/en/book.errorfunc.php">fatal error log</a>; will not contain stack overflows. Errors from this log are also shown in the Dashboard.</td>
       </tr>
```
Comment (ari-gold) [inline source/_docs/logs.md]: How about adding more rows to this table for other types of logs and removing some of the questions below?

Drupal database log | instructions on how to access
WordPress error log | instructions to enable/access
Varnish logs | not available
Solr logs | not available

**Comment by ari-gold** [comment]
Add to intro  at top of page? to give context that server logs are available via SFTP. Application-level logs can be accessed through Drupal directly...  In addition to logs, <New Relic> is another great way to help diagnose and fix errors and performance bottlenecks.

File: source/_docs/logs.md [hunk 3]
```diff
@@ -147,9 +147,78 @@ sh collect-logs.sh
...
+
+The client IP for the following example is `122.248.101.126`:
+
+```nginx
+10.x.x.x - - [19/Feb/2016:02:00:00 +0000]  "GET /edu HTTP/1.1" 200 13142 "https://pantheon.io/agencies/pantheon-for-agencies" "Mozilla/5.0 (Windows NT 6.3; WOW64; rv:43.0) Gecko/20100101 Firefox/43.0" 0.399 "122.248.101.126, 50.57.202.75, 10.x.x.x, 10.x.x.x"
+```
+
+#### Can I log to the system logger and access syslog?
+
+The short answer is no, syslog is not available. Technically, you can log Drupal events using the syslog module, but you won't be able to read or access them.  You can use the [error_log](http://php.net/manual/en/function.error-log.php) function to log to the php-error.log, which is accessible in the logs directory.
```
Comment (ari-gold) [inline source/_docs/logs.md]: Can we make this answer shorter, maybe starting by not calling it a short answer? =)

**Comment by ari-gold** [comment]
The "Enable Passwordless Access" seems abrupt/out of place. Change heading to more general? "Access logs via SFTP" and provide instructions to add ssh key as part of paragraph?

**Comment by ari-gold** [comment]
@nataliejeremy Like combining these docs! Overall looks great, just added a few suggestions to tighten things up since we're here.

**Pull request merged**